step/step.go

```diff
@@ -0,0 +1,445 @@
+package step
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+	"time"
+
+	androidcache "github.com/bitrise-io/go-android/cache"
+	"github.com/bitrise-io/go-android/gradle"
+	"github.com/bitrise-io/go-steputils/cache"
+	"github.com/bitrise-io/go-steputils/stepconf"
+	"github.com/bitrise-io/go-steputils/tools"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-io/go-utils/sliceutil"
+	"github.com/kballard/go-shellquote"
+)
+
+// Input ...
+type Input struct {
+	ProjectLocation string `env:"project_location,dir"`
+	AppPathPattern  string `env:"app_path_pattern,required"`
+	Variant         string `env:"variant"`
+	Module          string `env:"module"`
+	BuildType       string `env:"build_type,opt[apk,aab]"`
+	Arguments       string `env:"arguments"`
+	CacheLevel      string `env:"cache_level,opt[none,only_deps,all]"`
+	DeployDir       string `env:"BITRISE_DEPLOY_DIR,dir"`
+}
+
+// Config ...
+type Config struct {
+	ProjectLocation string
+
+	Variant string
+	Module  string
+
+	AppPathPattern string
+	AppType        AppType
+	Arguments      string
+
+	CacheLevel cache.Level
+	DeployDir  string
+}
+
+type AppType string
+
+const (
+	AppTypeAPK = AppType("apk")
+	AppTypeAAB = AppType("aab")
+)
+
+var (
+	ignoredSuffixes = [...]string{"Classes", "Resources", "UnitTestClasses", "AndroidTestClasses", "AndroidTestResources"}
+)
+
+const (
+	apkEnvKey     = "BITRISE_APK_PATH"
+	apkListEnvKey = "BITRISE_APK_PATH_LIST"
+
+	aabEnvKey     = "BITRISE_AAB_PATH"
+	aabListEnvKey = "BITRISE_AAB_PATH_LIST"
+
+	mappingFileEnvKey  = "BITRISE_MAPPING_PATH"
+	mappingFilePattern = "*build/*/mapping.txt"
+)
+
+type Result struct {
+	appFiles     []gradle.Artifact
+	appType      AppType
+	mappingFiles []gradle.Artifact
+}
+
+type ExportConfig struct {
+	DeployDir      string
+	AppPathPattern string
```

Comment:
unused property

Reply:
do you think we should keep the struct with a single property?

Reply:
Thanks for catching this, removed in latest commit.